tests/dataframe/test_sort.py

```diff
@@ -13,6 +14,12 @@
 ###
 
 
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
I think we should preferably call the context manager explicitly in our tests, or pass the fixture into tests when we want to use it -- we've had issues in the past where autouse makes it confusing for people debugging our unit test

Reply:
Makes sense, making the fixture explicit.